relay-server/src/actors/project_upstream.rs

```diff
@@ -58,7 +59,7 @@ impl UpstreamQuery for GetProjectStates {
     }
 
     fn path(&self) -> Cow<'static, str> {
-        Cow::Borrowed("/api/0/relays/projectconfigs/?version=3")
+        Cow::Borrowed("/api/0/relays/projectconfigs/?version=4&global_config=false")
```

Comment:
I think i'll revert this, as we have to wait first until the sentry side change is deployed. but this is how i think it will look later.